element/test/index.js

```diff
@@ -51,6 +51,21 @@ describe( 'element', () => {
 			) ).toBe( '<strong>Courgette</strong>' );
 		} );
 
+		it( 'should escape attributes and html', () => {
+			const result = renderToString( createElement( 'a', {
+				href: '/index.php?foo=bar&qux=<"scary">',
+				style: {
+					backgroundColor: 'red',
+				},
+			}, '&copy (sic) &#169; &copy; 2018 <"WordPress" & Friends>' ) );
+
+			expect( result ).toBe(
+				'<a href="/index.php?foo=bar&amp;qux=<&quot;scary&quot;>" style="background-color:red">' +
+				'&amp;copy (sic) &#169; &copy; 2018 &lt;"WordPress" &amp; Friends>' +
```

Comment:
It feels weird that the closing `>` is not automatically converted. I know this is not needed in the minimal requirements but how does the current editor work?  Should we be consistent?